demo/text-field-validators-demos.html

```diff
@@ -81,13 +115,11 @@ <h3>Asynchronous Custom Validator</h3>
     <p>Extend <code>Vaadin.TextFieldElement</code> to create your own custom element,
       then override the <code>checkValidity()</code> to make your asynchronous validation,
       and finally set the <code>invalid</code> flag appropriately in the callback.</p>
-
-    <vaadin-demo-snippet id="text-field-validators-demos-asynchronous-custom-validator" ignore-ie when-defined="vaadin-text-field">
+    <vaadin-demo-snippet id="text-field-validators-demos-asynchronous-custom-validator"
+      ignore-ie when-defined="vaadin-text-field">
       <template preserve-content>
-        <my-ajax id="ajax1" handle-as="json" url="https://vaadin.com/validate-element"></my-ajax>
-        <chemical-element-text-field label="Element Name"></chemical-element-text-field>
-        Type a valid Chemical Element such as Hydrogen, Oxygen, Carbon, etc.
-
+        <chemical-element-text-field label="Username"></chemical-element-text-field>
```

Comment:
This element should be renamed to something like `username-field` and also the class name from `ChemicalElement` to `UsernameField` because it doesn't deal with chemical elements anymore.